container/build-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "plugins": [
+    {
+      "name": "prometheus",
+      "archive": "https://github.com/headlamp-k8s/plugins/releases/download/v0.1.4/prometheus-0.0.1.tgz"
```

Comment:
We will need to update this when the PR is merged updating the plugin to mui5 and a new release of it is made.